pull_request_template

```diff
@@ -0,0 +1,42 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+
+## Backport
+YES | NO
+
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Related PRs
+List related PRs against other branches:
+
+branch | PR
+------ | ------
+planetscale/vitess | [link]()
+vitessio/vitess [link]()
```

Comment:
IMO there is no need for this. You just put the link to an issue or PR and GitHub markdown automatically turns it into a link, with hover hints.

Reply:
So, to clarify, the user will just put some links. Or even just `#34` -- GitHub will turn that into the correct link to issue 34.